linebot/version.go

```diff
@@ -11,13 +11,33 @@
 // WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
 // License for the specific language governing permissions and limitations
 // under the License.
-//
-// Don't edit this file directly. This file is generated by script/tag.sh .
 
 package linebot
 
-const version = "8.10.0"
+import (
+	"runtime/debug"
+	"strings"
+	"sync"
+)
+
+var (
+	sdkVersion     = "8.unknown"
+	sdkVersionOnce sync.Once
+)
 
 func GetVersion() string {
-	return version
+	// getting the version of line-bot-sdk-go should be done only once. Computing it repeatedly is meaningless.
+	sdkVersionOnce.Do(func() {
+		info, ok := debug.ReadBuildInfo()
+		if !ok {
+			return
+		}
+		for _, dep := range info.Deps {
+			if strings.Contains(dep.Path, "github.com/line/line-bot-sdk-go") {
```

Comment:
Not sure if "github.com/line/line-bot-sdk-go" will list in info.Deps, because it is not dependency for github.com/line/line-bot-sdk-go.

I do a roughly testing on my personal project, it seems only list all deps in go.mod not build itself.

Reply:
Oops. So, this can't see all the build information for the library's users...? I'll create my own project and check it out as well.
Thank you for confirming in your project.

Reply:
I checked by creating minimum library calling `debug.ReadBuildInfo` like this patch in line-bot-sdk-go. Please take a look at https://github.com/line/line-bot-sdk-go/pull/521#issuecomment-2562751933.

Reply:
Oh. Sorry, I try to verify in binary project. 
`debug.ReadBuildInfo` will failure on binary project, but works well on package.
Thank you for the confirmation.